src/main/java/com/itschool/eventmanagment/models/entities/Event.java

```diff
@@ -25,12 +26,10 @@ public class Event {
     private Long id;
     @Column
     private String name;
-    @Column (name = "event_date_time")
-    private LocalDateTime eventDateTime;
     @JsonDeserialize(using = LocalDateTimeDeserializer.class)
     @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @Column
-    private LocalDateTime dateTime;
+    @Column (name = "event_date_time")
+    private LocalDateTime eventDateTime;
     @Column
     private String location;
     @Column (name = "max_participants")
```

Comment:
It's not strictly necessary to include business logic methods like `registerParticipant()` in the entity itself. In many cases, you might move this kind of logic to a service class (e.g., EventServiceImpl) to follow the separation of concerns principle, which suggests that entities should mainly represent data and relationships, while business logic should be handled by services.

I could not comment on `registerParticipant()` method from this class, from line 50, so I made a coment on the property.